src/MyEventEmitter.js

```diff
@@ -1,12 +1,79 @@
 'use strict';
 
-export class MyEventEmitter {
-  on() {}
-  once() {}
-  off() {}
-  emit() {}
-  prependListener() {}
-  prependOnceListener() {}
-  removeAllListeners() {}
-  listenerCount() {}
+class MyEventEmitter {
+  constructor() {
+    this.events = new Map();
+  }
+
+  on(eventName, listener) {
+    if (!this.events.has(eventName)) {
+      this.events.set(eventName, []);
+    }
+    this.events.get(eventName).push(listener);
+  }
+
+  once(eventName, listener) {
+    const wrapper = (...args) => {
+      listener(args);
```

Comment:
Replace 'listener(args)' with 'listener(...args)' to spread the arguments properly.